xds/src/main/java/io/grpc/xds/internal/security/trust/XdsX509TrustManager.java

```diff
@@ -204,7 +215,12 @@ void verifySubjectAltNameInChain(X509Certificate[] peerCertChain) throws Certifi
   @Override
   public void checkClientTrusted(X509Certificate[] chain, String authType, Socket socket)
       throws CertificateException {
-    delegate.checkClientTrusted(chain, authType, socket);
+    if (delegates != null) {
```

Comment:
There seem to be several other "checkClientTrusted" and "checkServerTrusted" methods - don't we need to modify those as well?